composer.json

```diff
@@ -62,23 +42,64 @@
         "typo3/cms-scheduler": "dev-main as 12.0",
         "typo3/cms-seo": "dev-main as 12.0",
         "typo3/cms-setup": "dev-main as 12.0",
+        "typo3/cms-styleguide": "dev-main",
         "typo3/cms-sys-note": "dev-main as 12.0",
         "typo3/cms-t3editor": "dev-main as 12.0",
         "typo3/cms-tstemplate": "dev-main as 12.0",
         "typo3/cms-viewpage": "dev-main as 12.0",
         "typo3/cms-workspaces": "dev-main as 12.0",
-        "ttn/tea": "^3.0",
-        "t3docs/examples": "dev-main",
-        "t3docs/site-package": "dev-main",
-        "typo3-documentation-team/speeddemo": "^11.4",
-        "typo3/cms-styleguide": "dev-main"
+        "typo3/coding-standards": "^0.5.5"
+    },
+    "repositories": {
+        "1a_git": {
+            "type": "vcs",
+            "url": "https://github.com/TYPO3-Documentation/tea.git"
+        },
+        "1c_git": {
+            "type": "vcs",
+            "url": "https://github.com/TYPO3-Documentation/blog_example.git"
+        }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "config": {
+        "allow-plugins": {
+            "typo3/cms-composer-installers": true,
+            "typo3/class-alias-loader": true,
+            "ergebnis/composer-normalize": true
+        },
+        "bin-dir": ".Build/bin",
+        "sort-packages": true,
+        "vendor-dir": ".Build/vendor"
+    },
+    "extra": {
+        "typo3/cms": {
+            "extension-key": "t3docs-reference-coreapi",
+            "web-dir": ".Build/web"
+        }
     },
     "scripts": {
-        "generate:codesnippets": [
-            ".Build/vendor/bin/typo3 codesnippet:create Documentation/"
+        "check": [
+            "@check:php"
+        ],
+        "check:php": [
+            "@check:php:lint",
+            "@check:php:cs"
+        ],
+        "check:php:cs": "Build/Scripts/runTests.sh -n -s cgl",
+        "check:php:lint": "Build/Scripts/runTests.sh -s lint",
+        "fix": [
+            "@fix:php"
         ],
+        "fix:php": [
+            "@fix:php:cs"
+        ],
+        "fix:php:cs": "Build/Scripts/runTests.sh -s cgl",
         "generate": [
             "@generate:codesnippets"
+        ],
+        "generate:codesnippets": [
+            ".Build/vendor/bin/typo3 codesnippet:create Documentation/"
```

Comment:
So we can generate now directly from the repo?

Reply:
Noting changed here except the formating of the composer.json